### PR TITLE
Add option to have a Switch for ListItem

### LINKED ISSUE #256: ListItem Switch Component
It would be nice to have a Switch component available for the ListItem component for menus. Creating a PR to add this.

### PR DESCRIPTION
**Pull request by Joshuabaker2**
Resolves #256

### CODE REVIEW

File: src/list/ListItem.js [hunk 1]
```diff
@@ -213,6 +241,12 @@ styles = StyleSheet.create({
     alignItems: 'flex-end',
     justifyContent: 'center'
   },
+  switchContainer: {
+    flex: 0.15,
```
Comment (binoy14) [inline src/list/ListItem.js]: @Joshuabaker2 just curious why this specific number for flex?

**Comment by binoy14** [comment]
@Joshuabaker2 Sorry, but can you fix the merge conflicts. We are merging to `v0.11.0` branch

File: src/list/ListItem.js (hunk 1 above)
Comment (Joshuabaker2) [inline src/list/ListItem.js]: @binoy14 it was the number for the chevronContainer so I figured it wouldn't hurt to keep it uniform. Pretty arbitrary reason :)

Comment (binoy14) [inline src/list/ListItem.js]: @Joshuabaker2 All good. Once the merge conflict is fixed. I will merge it. Thanks again

**Comment by Joshuabaker2** [comment]
Merge conflicts fixed, let me know if you'd like me to squash as well!

**Comment by binoy14** [comment]
@Joshuabaker2 It's okay. Merging it now 🎊

**Comment by Joshuabaker2** [comment]
Thanks for all your help and being so responsive @binoy14!

**Comment by binoy14** [comment]
@Joshuabaker2 Thanks for the PR! Great stuff

**Pull request merged**